website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -511,25 +511,22 @@ this command.
 
 To list secrets for KV v2, a user must have a policy granting them the list capability on this `/metadata/` path - even if all the rest of their interactions with the KV v2 are via the `/data/` APIs. Access to at least list the `/metadata/` path should typically also be granted.
 
-| Method | Path                                 |
-|:-------|:-------------------------------------|
-| `LIST` | `/:secret-mount-path/metadata/:path` |
+| Method | Path                                      |
+|:-------|:------------------------------------------|
+| `GET`  | `/:secret-mount-path/metadata/?list=true` |
 
 ### Parameters
 
 - `secret-mount-path` `(string: <required>)` - The path to the KV mount containing
 the secret to list, such as `secret`. This is specified as part of the URL.
 
-- `path` `(string: <required>)` – Specifies the path of the secrets to list.
-  This is specified as part of the URL.
-
 ### Sample request
 
 ```shell-session
 $ curl \
     --header "X-Vault-Token: ..." \
-    --request LIST \
-    https://127.0.0.1:8200/v1/secret/metadata/my-secret
+    --request GET \
+    https://127.0.0.1:8200/v1/secret/metadata/?list=true
```

Comment:
It's not that using `LIST` is incorrect, rather the docs used a poor example with trying to list `my-secret`, which if we follow the earlier examples is a single secret, not a path to secrets. The docs say themselves:
> The input must be a folder; list on a file will not return a value.

The `GET` command works not because of the change in method, but because you removed `my-secret` from the path and are listing from the root directory. Just removing `my-secret` from the `LIST` command would have the same result, and I do think we should stick with using `LIST`

Reply:
But I don't know if that's the best solution here, because we probably want to show that you can list from a subdirectory path and not just from the root of the KV mount. Maybe we change all examples of `my-secret` to `secret-path/my-secret`, then the list here can be on `secret-path`? It would slightly complicate the other examples though. @schavis any thoughts on what would be best to remove this confusion?